tests/unit/corruption.cpp

```diff
@@ -17,8 +17,8 @@ BOOST_AUTO_TEST_CASE(test_header_corruption) {
 	/* corrupt record's header in blob and check that record isn't considered as corrupted (since data is correct)
 	 * and read is failed with -EINVAL
 	 */
-	config_wrapper config;
```

Comment:
Note that renaming `config_wrapper` -> `eblob_config_test_wrapper` is also refactoring. The other refactoring is modifying `eblob_config_test_wrapper` internals. And the 3rd action is that you add new tests. But I not insist on splitting that on different commits, because I'm very big-hearted, lololo =)